_episodes/11-using-arrays-transformations.md

```diff
@@ -13,41 +13,45 @@ keypoints:
 ---
 
 ## Arrays
-An 'Array' is a list of values, represented in Refine by the use of square brackets containing a list of values surrounded by inverted commas and separated by commas. For example an array listing the days of the week would look like:
+An 'Array' is a list of values, represented in OpenRefine by the use of square brackets containing a list of values surrounded by inverted commas (or double quotes) and separated by commas. For example an array listing the Subjects column would look like:
```

Comment:
Discussed rephrasing this para:
"An 'Array' is a data type (as mentioned [in the previous lesson](https://librarycarpentry.org/lc-open-refine/10-data-transformation/index.html)) which can contain a list of values. In OpenRefine an array is represented by the use of square brackets containing a list of values separated by commas.

For example:
* an array containing a list of strings (in this case subject keywords or phrases) could look like: `["crystal structure", "clozapinium", "crystal structure", "molecular configuration", "hydrogen bonding", "supramolecular assembly", "Chemistry", "QD1-999"]`
* an array containing a list of numbers could look like: `[1, 2, 3, 4]`

Reply:
Done  (may have lost the formatting)